themes/bootstrap3/templates/search/results.phtml

```diff
@@ -26,11 +24,7 @@
   );
 
   // Set up breadcrumbs:
-  if (isset($this->overrideTitle)) {
-    $this->layout()->breadcrumbs .= '<li class="active">' . $this->escapeHtml($this->overrideTitle) . '</li>';
-  } else {
-    $this->layout()->breadcrumbs .= '<li class="active">' . $this->transEsc('Search') . ': ' . $this->escapeHtml($lookfor) . '</li>';
```

Comment:
Same comment as above: this change will make the breadcrumbs longer. Not sure if that matters, but want to be sure you're aware.